cmd/kyma/provision/gardener/cmd.go

```diff
@@ -139,7 +149,7 @@ func newProvider(o *Options) (*types.Provider, error) {
 		v := strings.Split(e, "=")
 
 		if len(v) != 2 {
-			return p, fmt.Errorf("Wrong format for extra configuration %s. Please provide NAME=VALUE pairs.", e)
```

Comment:
Is this `e` deleted by mistake? If not, please adapt the error message as it has a formatting variable `%s` in it.

Reply:
That would be a mistake, let me fix it.